docs/_docs/Developers/build.md

```diff
@@ -5,45 +5,91 @@ order: 1
 toc: true
 ---
 
-## Ubuntu 18.04/18.10 and Linux Mint 19
+## Ubuntu 18.04/18.10
+*Updated by [kreyren](https://github.com/Kreyren) at 24.02.2019*
 
-* Install the dependencies:
-```
+Fork https://github.com/PhoenicisOrg/phoenicis on github before building.
+
+#### Dependencies:
+- git
+- maven
+- java11+
+- fakeroot
+
+#### Building:
+```bash
+# Update repository 
 sudo apt update
+
+# Install dependencies
 sudo apt install git maven openjdk-11-jdk fakeroot
-```
 
-* Optional: if your distribution does not support Java 10+ **or** if you need to create packages, grab the latest JDK 
-```
- wget https://download.java.net/java/GA/jdk11/28/GPL/openjdk-11+28_linux-x64_bin.tar.gz -O /tmp/openjdk-11+28_linux-x64_bin.tar.gz
- sudo tar xfvz /tmp/openjdk-11+28_linux-x64_bin.tar.gz --directory /usr/lib/jvm
- export JAVA_HOME="/usr/lib/jvm/jdk-11"
-```
- 
+# Clone your phoenicis repository
+git clone https://github.com/<YOUR_GITHUB_FORK_REPO>/phoenicis.git
 
-* Clone the repository from GitHub:
-```
-git clone https://github.com/PhoenicisOrg/phoenicis.git
-```
+# Sanity-check your JAVA_HOME variable
+echo JAVA_HOME
+export JAVA_HOME=PATH # if needed, should be set automatically.
 
-* Set the Java version:
-```
-export JAVA_HOME=/usr/lib/jvm/java-11-openjdk-amd64
+# Build phoenicis
+mvn clean package
+
+# Create packages
+cd phoenicis-dist/src/scripts
+bash phoenicis-create-package.sh
 ```
 
-* Build Phoenicis:
+#### OPTIONAL: Copypasta method
```

Comment:
What's the point of this part?

Reply:
@plata i'm lazy and i like copy+paste

Reply:
I believe @plata wants to point out that the relation between the header and the content of the mentioned section is unclear and needs some improvement. `Copypasta method` doesn't really describe what the commands do and why a user should use them.

Reply:
And actually this code won't even work for anybody else then you (there's several "Kreyren" in it). So this is nothing we can merge to an official tutorial.

Reply:
@plata fair enough, fixing

Reply:
Fixed in https://github.com/PhoenicisOrg/phoenicis/pull/1860/commits/66e67403130cf212ea576a4a973a2d7465da5dc1